test/app/components/sub_component_delegate_component.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class SubComponentDelegateComponent < ViewComponent::Base
```

Comment:
Add comment about this component using a component from another file as a subcomponent/slot?